free-programming-books-zh.md

```diff
@@ -76,6 +76,7 @@
   * [Vim](#vim)
   * [Visual Prolog](#visual-prolog)
   * [WebAssembly](#webassembly)
+  * [TypeScript](#typescript)
 
```

Comment:
At first, you'd better to read the [CONTRIBUTING.md
](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)   

for [TypeScript] menu you should add it ref below image   

All the lists start with an Index. The idea is to list and link all sections and subsections there. Keep it in **alphabetical order.**

![image](https://user-images.githubusercontent.com/25427352/52986290-3002a680-3432-11e9-830d-4cff863732c4.png)